src/server/system_services/schemas/nsfs_bucket_schema.js

```diff
@@ -59,6 +59,9 @@ module.exports = {
         },
         website: {
             $ref: 'common_api#/definitions/bucket_website',
+        },
+        force_md5_etag: {
+            type: 'boolean',
```

Comment:
Since it is a schema change suggesting to add a test in the file `test_nc_nsfs_bucket_schema_validation.test.js`.
For example:
https://github.com/noobaa/noobaa-core/blob/d1bf8c465af745dedbff9d3803ec13da344884ba/src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_schema_validation.test.js#L56-L60